frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -146,48 +163,26 @@ export const insightLogic = kea<insightLogicType>({
                 setInsightMode: (_, { source }) => source,
             },
         ],
-    },
-    loaders: ({ values, actions }) => ({
-        insight: {
-            __default: { tags: [] } as Partial<DashboardItemType>,
-            loadInsight: async (id: number) => await api.get(`api/insight/${id}`),
-            updateInsight: async (payload: Partial<DashboardItemType>, breakpoint) => {
-                if (!Object.entries(payload).length) {
-                    return
-                }
-                await breakpoint(700)
-                return await api.update(`api/insight/${values.insight.id}`, payload)
-            },
-            setInsight: (insight) => insight,
-        },
         insightMode: [
             ItemMode.View as ItemMode,
             {
-                setInsightMode: ({ mode }) => {
-                    if (mode === ItemMode.Edit) {
-                        clearDOMTextSelection()
```

Comment:
I wonder if this is still needed? Are we keeping the same behavior of the editing toast or not anymore?